app/graphql/feature-flags-context.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, createContext, useContext, useEffect } from "react"
```

Comment:
it's not really a graphql related feature?

Reply:
looking at the filename

Reply:
Any suggestions on where to put it? Maybe under the `config` folder or just the components.